### ISSUE TITLE
Build tensorflow classifiers

### ISSUE THREAD

**Issue by stevehadd** [Fracappo87/XBTs_classification#39]
We're currently running in to a limitatin of scikit-learn when we want to deploy the trained model. One can pickle the classifier object, but this is not a robust way to save the trained model. Other packages have better options for saving a trained model built in to the library, for example tensorflow.  Tensor flow does not have a decision tree classifier, but does have a Gradient Boosted trees and random forest classifiers, which have shown similar results to DTs in scikit learn. We should thus try to move to using tensorflow (or other library potentially e.g. pyTorch, azure ML, AWS sagemaker etc.)
